feature/home/src/main/java/com/serranoie/app/feature/home/HomeScreen.kt

```
@@ -1002,145 +1004,269 @@ fun SummarySection(
-                    }
+                                overview.yesterdayExpenses.forEach { expense ->
+                                    Row(
+                                        modifier = Modifier.fillMaxWidth(),
+                                        horizontalArrangement = Arrangement.SpaceBetween
+                                    ) {
+                                        Text(
+                                            text = "${expense.name} — ${expense.paidBy.name} ${expense.paidBy.surname}".trim(),
+                                            style = MaterialTheme.typography.bodyMedium
+                                        )
+                                        Text(
+                                            text = "$${String.format("%.2f", expense.amount)}",
```

detekt (1.23.8)

[warning] String.format("%.2f", expense.amount) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)